tendrl/commons/event.py

```diff
@@ -0,0 +1,25 @@
+from gevent import socket
+from gevent.socket import error as socket_error
+from gevent.socket import timeout as socket_timeout
+from tendrl.commons.message import Message
+
+
+class EventFailed(Exception):
+    pass
+
+
+class Event(object):
+    def __init__(self, message):
+        self.sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+        self._write(message)
+
+    def _write(self, message):
+        try:
+            socket_path = tendrl_ns.config.data['logging_socket_path']
+            self.sock.connect(socket_path)
```

Comment:
@nthomas-redhat socket is created only once, which is return in node agent, here event class will connect with socket which is running already and pass send the message into it